EntAppProject/src/main/java/com/jobook/EntAppProject/Services/JobServiceStub.java

```diff
@@ -1,58 +1,60 @@
-import org.springframework.stereotype.Component;
-
-import com.jobook.EntAppProject.model.Customer;
-import com.jobook.EntAppProject.model.Employee;
-import com.jobook.EntAppProject.model.Job;
-
-@Component
-public class JobServiceStub implements IJobService {
-	
-	@Autowired
-	IUserService userService;
-	
-	private List<Job> jobs; 
-	
-	private void generateTestJobs()
-	{
-		jobs = new ArrayList<Job>();
-		Customer customer = (Customer) userService.getUserInfoById(0);
-		Employee employee = (Employee) userService.getUserInfoById(1);
-		jobs.add(new Job(0, customer.getId(), null, customer, null, "This is Test Job 1.", "Test Job 1", null, null, 100.0));
-		jobs.add(new Job(1, customer.getId(), employee.getId(), customer, employee, "This is Test Job 2.", "Test Job 2", null, null, 300.0));
-		jobs.add(new Job(2, null, employee.getId(), null, employee, "This is Test Job 3.", "Test Job 3", null, null, 500.0));
-		
-	}
-	
-	public List<Job> getUsersRelatedJobs(int userId) {
-		generateTestJobs();
-		List<Job> filteredJobs = new ArrayList<Job>();
-		
-		for (Job job : jobs)
-		{
-			if ((job.getCustomerId() != null && job.getCustomerId() == userId) || (job.getEmployeeId() != null && job.getEmployeeId() == userId))
-				filteredJobs.add(job);
-		}
-		
-		return filteredJobs;
-	}
-
-	public List<Job> searchJobByName(String searchTerm) {
-		generateTestJobs();
-		List<Job> filteredJobs = new ArrayList<Job>();
-		
-		for (Job job : jobs)
-		{
-			if (job.getShortDesc().contains(searchTerm))
-				filteredJobs.add(job);
-		}
-		
-		return filteredJobs;
-	}
-
-}
+package com.jobook.EntAppProject.Services;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+import com.jobook.EntAppProject.model.Customer;
+import com.jobook.EntAppProject.model.Employee;
+import com.jobook.EntAppProject.model.Job;
+
+@Component
+public class JobServiceStub implements IJobService {
+	
+	@Autowired
+	IUserService userService;
+	
+	private List<Job> jobs; 
+	
+	private void generateTestJobs()
+	{
+		jobs = new ArrayList<Job>();
+		Customer customer = (Customer) userService.getUserInfoById(0);
+		Employee employee = (Employee) userService.getUserInfoById(1);
+		jobs.add(new Job(0, customer.getId(), null, customer, null, "This is Test Job 1.", "Test Job 1", null, null, 100.0));
+		jobs.add(new Job(1, customer.getId(), employee.getId(), customer, employee, "This is Test Job 2.", "Test Job 2", null, null, 300.0));
+		jobs.add(new Job(2, null, employee.getId(), null, employee, "This is Test Job 3.", "Test Job 3", null, null, 500.0));
+		
+	}
+	
+	public List<Job> getUsersRelatedJobs(int userId) {
+		generateTestJobs();
+		List<Job> filteredJobs = new ArrayList<Job>();
+		
+		for (Job job : jobs)
+		{
+			if ((job.getCustomerId() != null && job.getCustomerId() == userId) || (job.getEmployeeId() != null && job.getEmployeeId() == userId)) {
```

Comment:
When using a singular if statement that will determine if one line of code will run, it can be tempting to avoid using the `{`. However, I usually recommend that you put them in anyway just in case later down the road you decide to add more statements as a result of that logical expression, you do not want to forget to re-add the `{`.

Reply:
I agree... good point.